### ISSUE TITLE
Keyword Search for RDF Elasticsearch 

### ISSUE THREAD

**Issue by jermnelson** [LD4P/sinopia_editor#631]
For the MVP we need to support a basic keyword search of the RDF Elasticsearch Index for both the `InputLookupSinopia` React component as a typeahead search and the Browse tab for finding existing RDF entities to load and edit in the Editor.

Blocks #442 
~~Blocked by #630~~
~~Blocked by https://github.com/LD4P/sinopia_indexing_pipeline/issues/25~~

**Comment by mjgiarlo** [comment]
@jermnelson What work does this issue cover that isn't already covered by https://github.com/LD4P/sinopia_editor/issues/630, https://github.com/LD4P/sinopia_indexing_pipeline/issues/25, or https://github.com/LD4P/sinopia_editor/issues/442? I'm not sure what is needed to close this issue.

**Comment by jermnelson** [comment]
I think the titles and subtitles indexing https://github.com/LD4P/sinopia_indexing_pipeline/issues/25 should be sufficient for this issue. Closing.
